src/server/req_quejob.c

```diff
@@ -3325,9 +3285,7 @@ long long get_next_svr_sequence_id(void)
  */
 void reset_svr_sequence_window(void)
 {
-	svr_sequence_window_count = 0;
 	server.sv_qs.sv_jobidnumber = 0;
```

Comment:
Since this function got reduced to one line, do we still need it?

Reply:
I will leave it in a function, lest tomorrow we decide to add some more logic to it, like walk over and try to find a free slot first